app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationItemFactory.kt

```diff
@@ -97,11 +98,19 @@ private fun GeneralConversationItem(
                     eventType = conversation.badgeEventType,
                     clickable = onConversationItemClick,
                     trailingIcon = {
-                        if (hasOnGoingCall)
-                            JoinButton(buttonClick = onJoinCallClick)
-                        else if (mutedStatus != MutedConversationStatus.AllAllowed) {
-                            MutedConversationBadge(onMutedIconClick)
-                            HorizontalSpace.x8()
+                        Row {
+                            if (hasOnGoingCall) {
+                                HorizontalSpace.x8()
+                                JoinButton(buttonClick = onJoinCallClick)
+                                HorizontalSpace.x8()
+                            }
+                            else if (mutedStatus != MutedConversationStatus.AllAllowed) {
+                                HorizontalSpace.x8()
+                                MutedConversationBadge(onMutedIconClick)
+                                HorizontalSpace.x8()
+                            } else {
+                                HorizontalSpace.x48()
```

Comment:
Why exactly 48? Muted badge has width 24dp and join button minimum 60dp so I'm not sure, does this value come from the designs? 🤔

Reply:
On designs, without any button or icon, texts end 39 from the right edge so maybe set the padding to 40?
<img width="543" alt="image" src="https://user-images.githubusercontent.com/30429749/209945898-dd107f96-21e1-4818-80ec-09285f6d9283.png">

Reply:
It's little bit tricky one, I think designs should be updated to have only 16 pixels padding like from left side when there is no badge